### DISCUSSION TITLE
Add ability to set Z height offset

### DISCUSSION THREAD

**Discussion by shughey00** [OrcaSlicer/OrcaSlicer#1075] [Ideas]
I realize this can be done via g code but that annoying. Would be nice to have a menu item to set Z height offset to fine tune first layer.

**Comment by hliebscher** [comment]
you can set the offset in GCode .. for tuning you have to use babysteps in printer menu during print and save this

**Comment by imagesurgery** [comment]
I too would like this feature. Combined with per-object settings, this would allow me to automate my currently manual gcode editing process, and be able to level and test that level with far fewer manual points of failure, with different nozzles or layer heights, etc. to ensure a bombproof first layer. Editing gcode is not ideal when even a simple 9x9 test grid can run in the mid tens of thousands of lines. FYI my printer bed is huge, so levelling is insanely important to successful prints.

**Comment by tripletto** [comment]
I'd like / need this as well

**Comment by Andychud** [comment]
Yes, please add this feature.

**Comment by Egosumumbravir** [comment]
This is quite possibly the only Cura feature I miss in Orca Slicer.
Possibly best in the filament options?  Some plastics need a bit more squish, some need a bit less. Once you've worked it out it's a pain to keep babystepping on a console when switching filaments.

**Reply by spitfire** [reply]
For me it would be more depending on the type of plate I'm using - on some smooth ones I'd like more squish, on the other ones (and the textured ones) not. That could be independent from either filament or plate and in general (Strength?) settings.

**Comment by rilkerfc** [comment]
Will be great feature

**Comment by oscarwhk** [comment]
Yes, I want it too.

I am using Bambu Lab A1 mini, my problem is the initial print head location is too close to the heat plate.

**Reply by spitfire** [reply]
And I would like it to do the opposite - make it a little closer to the plate so the first layer is squished for better adhesion and nicer surface of the print. Which is what would make this feature great. I remember having this a a plugin in Cura - it would let me set a z-offset for a whole print, and an additional one just for the first layer. That would be so great to have here :)

**Comment by Sangra6969** [comment]
I don't think it is working properly.  I even used BED_MESH_CLEAR before every print, and it drags the nozzle on the bed. When I set Klipper to .2 everything is fine. When Orca does it, it doesn't feel like it moves the Z.
